framework/source/class/qx/ui/core/scroll/AbstractScrollArea.js

```diff
@@ -267,9 +257,10 @@ qx.Class.define("qx.ui.core.scroll.AbstractScrollArea",
     */
 
     /**
-     * Returns the boundaries of the pane.
+     * Returns the dimensions of the pane.
      *
-     * @return {Map} The pane boundaries.
+     * @return {{width: number, height: number}|null} The pane dimension in pixel. Contains
```

Comment:
This works well with my code editor (PhpStorm and by extension WebStorm from Jetbrains) which has Qooxdoo support build in. I think it uses JSDoc (http://usejsdoc.org/) to interpret the doc-blocks.